app/controllers/groups_controller.rb

```diff
@@ -102,6 +116,15 @@ def authorized_namespaces
                                               type: :relation, as: :manageable).where.not(type: Namespaces::UserNamespace.sti_name) # rubocop:disable Layout/LineLength
   end
 
+  def subgroups_and_projects(id)
+    @subgroups_and_projects = Namespace.where(
```

Comment:
**nitpick (blocking):** Please move the query to `Namespace` model as a new method named `children_of_type` which accepts an argument `type` and then uses the query here without order and without `include_route`.

Reply:
Updated in 38305b2